tools/log-collector/helper.go

```diff
@@ -153,62 +148,37 @@ func filterCSV(csvObjects unstructured.UnstructuredList) unstructured.Unstructur
 // filterCRD returns list of crds created by given set of groups
 func filterCRD(crdObjs unstructured.UnstructuredList) (unstructured.UnstructuredList, error) {
```

Comment:
rename func. just `filterCRD` does not make any sense

Reply:
resolved